### DISCUSSION TITLE
Accessibility: Incorrect focus behaviour on modal

### DISCUSSION THREAD

**Discussion by dahalsaur** [add2cal/add-to-calendar-button#295] [Ideas / Requests]
1. Focus is not directed to the first functional element of the modal window
When you open "Add to calendar" modal using keyboard, you can see that the focus is not at the first functional element of the modal window. In the screenshot, it  should have focused on "Apple" element on clicking "Add to calendar".

2. Focus is not trapped within the modal window
The tab navigation is escaping out of the modal, which is not the correct behaviour. When you navigate through elements in the modal using tab, it should always loop through modal elements unless user clicks esc or the close button.

Tools used: Google Chrome Version 108.0.5359.124 (Official Build) (arm64) using keyboard navigation

![Screenshot 2023-01-16 at 09 06 43](https://user-images.githubusercontent.com/39714407/212627864-983ab58c-ae20-4baa-a7c5-363b92648c16.png)

**Comment by jekuer** [comment]
Should be optimized with the latest V2 version 😃
